libsolidity/ast/ASTJsonImporter.cpp

```diff
@@ -25,6 +25,7 @@
 
 #include <libyul/AsmJsonImporter.h>
 #include <libyul/AsmParser.h>
+#include <libyul/AST.h>
```

Comment:
Is it needed in this file (as there was no include before here)?

Reply:
Previously `ASTJsonImporter.h` included it, but this PR changes that to only include the forward declaration.

Reply:
Ok, got it. Thanks